### PR TITLE
Docs: Initial docs for the annotation panel launching in 8.1

### PR DESCRIPTION
**Pull request by pkolyvas**
**What this PR does / why we need it**: Initial documentation for the new annotation panel in 8.1

**Special notes for your reviewer**: Could use some focus on the Time before / time after section.

### CODE REVIEW

File: docs/sources/panels/visualizations/annotations.md [hunk 1]
```diff
@@ -6,6 +6,65 @@ aliases =["/docs/grafana/latest/features/panels/anotations/"]
 weight = 600
 +++
 
-# Annotations
+# Annotations 
 
-The Annotations panel visualization allows you to view ...
\ No newline at end of file
+The Annotations panel allows you to create lists of annotations that can be used to view annotated data. Various options are available to filter the list based on tags and the current dashboard.
```
Comment (ryantxu) [inline docs/sources/panels/visualizations/annotations.md:11]:
```suggestion
The Annotations panel allows you to show a list of annotations that can be used to view annotated data. Various options are available to filter the list based on tags and the current dashboard.
```

File: first-pass.csv [hunk 2]
```diff
@@ -0,0 +1,290 @@
+21301,OPEN
```
Comment (ryantxu) [inline first-pass.csv:1]: accidental commit?

File: docs/sources/panels/visualizations/annotations.md [hunk 3]
```diff
@@ -6,6 +6,65 @@ aliases =["/docs/grafana/latest/features/panels/anotations/"]
...
+### Link target
+
+Use this option to chose how to view the annotated data.
+
+* Panel - This option will take you directly to a full-screen view of the panel with the corresponding annotation
+* Dashboard - This option will focus the annotation in the context of a complete dashboard
+
+### Time before
+
+Use this option to set the time range before the annotation.
```
Comment (ryantxu) [inline docs/sources/panels/visualizations/annotations.md:66]:
```suggestion
Use this option to set the time range before the annotation.  Use duration string values like "1h" = 1 hour, "10m" = 10 minutes, etc
```

Comment (ryantxu) [reply docs/sources/panels/visualizations/annotations.md:66]: I'm not sure if we describe this format anywhere else

**Review by ryantxu — COMMENTED** [review]
content looks good.

**Review by ryantxu — APPROVED** [review]

File: first-pass.csv (hunk 2 above)
Comment (pkolyvas) [reply first-pass.csv:1]: yep will remove

File: docs/sources/panels/visualizations/annotations.md [hunk 4]
```diff
@@ -6,6 +6,65 @@ aliases =["/docs/grafana/latest/features/panels/anotations/"]
...
+Use the query filter to create a list of annotations from all dashboards in your organization or the the current dashboard in which this panel is located.
+
+* All dashboards - List annotations from all dashboards in the current organization
+* This dashboard - Limit the annotions list to annotations on the same dashboard in which the annotation list panel appears
+
+### Time Range
+
+Use the time range option to select if the list should limited to the current time range.
+
+* None - no timerange limit for the annotations query
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/annotations.md:28]:
```suggestion
* None - no time range limit for the annotations query.
```

File: docs/sources/panels/visualizations/annotations.md [hunk 5]
```diff
@@ -6,6 +6,65 @@ aliases =["/docs/grafana/latest/features/panels/anotations/"]
...
+* This dashboard - Limit the annotations list to the timerange of the dashboard in which the annotation list pannel appears
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/annotations.md:29]:
```suggestion
* This dashboard - Limit the list to the time range of the dashboard where the annotation list panel is available.
```

File: docs/sources/panels/visualizations/annotations.md [hunk 6]
```diff
@@ -6,6 +6,65 @@ aliases =["/docs/grafana/latest/features/panels/anotations/"]
...
+
+### Tags
+
+Use the tags list to filter the annotations by tags. You can add multiple tags in order to refine the list.
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/annotations.md:33]:
```suggestion
Use the tags option to filter the annotations by tags. You can add multiple tags in order to refine the list. 
```

File: docs/sources/panels/visualizations/annotations.md [hunk 7]
```diff
@@ -6,6 +6,65 @@ aliases =["/docs/grafana/latest/features/panels/anotations/"]
...
+Use the time range option to select if the list should limited to the current time range.
+
+* None - no timerange limit for the annotations query
+* This dashboard - Limit the annotations list to the timerange of the dashboard in which the annotation list pannel appears
+
+### Tags
+
+Use the tags list to filter the annotations by tags. You can add multiple tags in order to refine the list. 
+
+> **Note:** Alternatively, you can leave the tag list empty and filter on-the-fly by selecting tags which are listed as part of the results on the panel itself.
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/annotations.md:35]:
```suggestion
> **Note:** Optionally, leave the tag list empty and filter on the fly by selecting tags that are listed as part of the results on the panel itself.
```

File: docs/sources/panels/visualizations/annotations.md [hunk 8]
```diff
@@ -6,6 +6,65 @@ aliases =["/docs/grafana/latest/features/panels/anotations/"]
...
+
+### Limit
+
+Use the limit option to limit the number of results returned.
+
+## Display
+
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/annotations.md:42]: Need content

File: docs/sources/panels/visualizations/annotations.md [hunk 9]
```diff
@@ -6,6 +6,65 @@ aliases =["/docs/grafana/latest/features/panels/anotations/"]
...
+### Show user
+
+Use this option to show or hide which user created the annotation. 
+
+### Show time
+
+Use this option to show or hide the time the annotation was created
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/annotations.md:49]:
```suggestion
Use this option to show or hide the time the annotation creation time.
```

File: docs/sources/panels/visualizations/annotations.md [hunk 10]
```diff
@@ -6,6 +6,65 @@ aliases =["/docs/grafana/latest/features/panels/anotations/"]
...
+
+### Show Tags
+
+Use this option to show or hide the tags associated with a given annotation. *NB*: You can use the tags to live-filter the annotation list on the panel itself.
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/annotations.md:53]:
```suggestion
Use this option to show or hide the tags associated with an annotation. *NB*: You can use the tags to live-filter the annotation list on the panel itself.
```

File: docs/sources/panels/visualizations/annotations.md [hunk 11]
```diff
@@ -6,6 +6,65 @@ aliases =["/docs/grafana/latest/features/panels/anotations/"]
...
+
+## Link behavior
+
+### Link target
+
+Use this option to chose how to view the annotated data.
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/annotations.md:59]:
```suggestion
Use this option to chose how to view the annotated data. It has the following options.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

File: first-pass.csv (hunk 2 above)
Comment (pkolyvas) [reply first-pass.csv]: Removed!

File: docs/sources/panels/visualizations/annotations.md (hunk 8 above)
Comment (pkolyvas) [reply docs/sources/panels/visualizations/annotations.md:42]: Added - thanks for the catch!

**Pull request merged**